### ISSUE TITLE
[BUG] backtrader package not working with matplotlib version 3.4.1

### ISSUE THREAD

**Issue by madca03** [enzoampil/fastquant#364]
#### Problem description
backtrader package not working with matplotlib version 3.4.1

### Solution 1
- update requirements.txt to use the latest version of backtrader in its master branch

### Solution 2
- reinstall backtrader to get the latest patch from the master branch
- `pip install git+https://github.com/mementum/backtrader.git@0fa63ef4a35dc53cc7320813f8b15480c8f85517#egg=backtrader`

### Solution sources
- `https://stackoverflow.com/questions/63471764/importerror-cannot-import-name-warnings-from-matplotlib-dates`

**Comment by mikeejazmines** [comment]
Yes I've also came across this issue multiple times, I've gone the path of running matplotlib 3.2.2 version
